index.yaml

```diff
@@ -1116,8 +1116,8 @@ entries:
     sources:
     - https://github.com/open-policy-agent/gatekeeper.git
     urls:
-    - assets/rancher-gatekeeper/rancher-gatekeeper-3.3.1+up3.3.0.tgz
-    version: 3.3.1+up3.3.0
+    - assets/rancher-gatekeeper/rancher-gatekeeper-3.3.001.tgz
+    version: 3.3.001
```

Comment:
Nothing actually changed with gatekeeper; the ordering was just flipped when regenerating the index.yaml. See below for the exact change applied to the original location of `3.3.1+up3.3.0`.